cron.py

```diff
@@ -1106,6 +1106,18 @@ def notify_hibp():
         Session.commit()
 
 
+def clear_users_scheduled_to_be_deleted():
+    users = User.filter(
+        and_(User.delete_on.isnot(None), User.delete_on < arrow.now())
+    ).all()
+    for user in users:
+        LOG.i(
+            f"Scheduled deletion of user {user} with scheduled delete on {user.delete_on}"
+        )
+        user.delete(user.id)
```

Comment:
this should rather be `Session.delete(user)` right?

Reply:
It's the same. I did it the same way as the job runner does https://github.com/simple-login/app/blob/master/job_runner.py#L222

Reply:
In this one it's `User.delete` (note that uppercase U)

Reply:
good one! Then how the ** do the test work :thinking: ?

Reply:
ah, bc it's a static method and the user object calls it regardless.. Changed it.

Reply:
Oh that's true.